### PR TITLE
trigs.cpp: Fix build error

### PR DESCRIPTION
**Pull request by glebm**

### CODE REVIEW

File: Source/levels/trigs.cpp [hunk 1]
```diff
@@ -754,33 +754,41 @@ bool ForcePWaterTrig()
 
 bool ForceArenaTrig()
 {
-	int *checkList = nullptr;
+	const uint16_t *checkList = nullptr;
+	size_t len;
```
Comment (qndel) [inline Source/levels/trigs.cpp:758]:
```suggestion
	size_t len = 0;
```
just to give it a default value if we did it with checkList? :P

Comment (glebm) [reply Source/levels/trigs.cpp:758]: No need because we always assign it (or `return false`)

Comment (glebm) [reply Source/levels/trigs.cpp:758]: Removed the default value for `checkList` as well

**Review by ephphatha — APPROVED** [review]

File: Source/levels/trigs.cpp (hunk 1 above)
Comment (qndel) [reply Source/levels/trigs.cpp:758]: that works too, just wanted consistency :p

**Pull request merged**